frontend/src/views/MovieAddScreen.vue

```diff
@@ -121,7 +121,7 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies-${movie.id}-upload-image/`, data, imageConfig);
```

Comment:
are you sure you have this endpoint on backend? upload please screenshot where you upload an image
also check the note from readme
<img width="858" alt="image" src="https://github.com/user-attachments/assets/578bc906-a04d-4058-baf5-b3542c0ecd35">